preload.js

```diff
@@ -362,6 +362,13 @@ setInterval(() => {
   window.nodeSetImmediate(() => {});
 }, 1000);
 
+// Set server-client time difference
+(async function setClockParams() {
```

Comment:
This seems like it would be better suited to be placed in `background.js` instead of `preload.js`. Preload is where you link node with the front end window.

Reply:
Also don't forget to run `yarn lint`